src/st_carousel.c

```diff
@@ -150,23 +170,42 @@ static void DrawIcon(int x, int y, sbarelem_t *elem, weapon_icon_t icon)
     }
 }
 
+static int CalcOffset(void)
+{
+    if (direction)
+    {
+        const int delta = I_GetTimeMS() - last_time;
+
+        if (delta < 125)
+        {
+            const float x = 1.0f - delta / 125.0f;
+            return lroundf(64.0f * x * x) * direction;
```

Comment:
Just an experiment:

https://github.com/user-attachments/assets/938f6b07-3279-4a60-a34b-bcf36966f89c

Reply:
I like it a lot!